pkg/cmd/roachtest/tests/pg_regress.go

```diff
@@ -432,6 +432,10 @@ func runPGRegress(ctx context.Context, t test.Test, c cluster.Cluster) {
 	if err != nil {
 		t.L().Printf("Failed to read %s: %s", testdata, err)
 	}
+
+	// Replace specific versions in issue URIs with a generic "_version_".
+	issueURI := regexp.MustCompile(`https:\/\/go\.crdb.dev\/issue-v\/(\d+)\/.*`)
```

Comment:
do you need to escape the . in crdb.dev?

Reply:
Also, should we restrict a bit more what's allowed for the version number? maybe not '/' and not ' '?